src/services/movies.service.js

```diff
@@ -15,10 +16,12 @@ const MoviesService = {
     return ApiService.get(moviesUrl.getAll);
   },
 
-  getCurrentPage(page) {
+  getCurrentPage(page, filter) {
     let currentUrl = moviesUrl.currentUrl
       .replace("%page", page)
-      .replace("%size", PER_PAGE);
+      .replace("%size", PER_PAGE)
+      .replace("%title", filter.title)
+      .replace("%genreId", filter.genreId);
```

Comment:
Jel moguce da neki nema? Nece biti problem u url-u?

Reply:
S obzirom da su sve ista imena, mogao si i:
```
function getQueryParams(filter) {
const searchParams = new URLSearchParams();
Object.keys(filter).forEach((key) => {
   searchParams.append(key, filter[key]);
});
return searchParams.toString();
}
```
dok god su isti, ali aj nije bas sjajna metoda samo ti onda sama vratila sve keyeve kako treba

Reply:
moguce je da nema ali onda ako posaljemo "title=&genreId=" u requestu, filter na be ce ignorisati to sto je prazno.